app/code/Magento/Catalog/Model/Category.php

```diff
@@ -977,12 +977,29 @@ public function getProductCount()
      * @param bool $sorted
      * @param bool $asCollection
      * @param bool $toLoad
+     * @param bool $onlyActive
+     * @param bool $includeMenu
      * @return \Magento\Framework\Data\Tree\Node\Collection|\Magento\Catalog\Model\ResourceModel\Category\Collection
      */
-    public function getCategories($parent, $recursionLevel = 0, $sorted = false, $asCollection = false, $toLoad = true)
-    {
-        $categories = $this->getResource()->getCategories($parent, $recursionLevel, $sorted, $asCollection, $toLoad);
-        return $categories;
+    public function getCategories(
+        $parent,
+        $recursionLevel = 0,
+        $sorted = false,
+        $asCollection = false,
+        $toLoad = true,
+        $onlyActive = true,
+        $includeMenu = true
+    ) {
+        return $this->getResource()
+            ->getCategories(
+                $parent,
+                $recursionLevel,
+                $sorted,
+                $asCollection,
+                $toLoad,
+                $onlyActive,
+                $includeMenu
```

Comment:
Hello, thank you for your contribution.

Reading the parameter name `$includeMenu` I was led to interpret the code as:
- `true` - select **only** categories that are **included** in the menu
- `false` - select **only** categories that are **not included** in the menu

Reading the code, instead, the real meaning of the two cases is:
- `true` - select **only** categories that are **included** in the menu (same as above)
- `false` - select **all categories**, regardless of the fact they are included in the menu or not

Doesn't `$onlyIncludeInMenu` sound more appropriate?

Cheers.

Reply:
Hi, @aleron75. Oh, why not - great. Thanks.